griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
+    secret_arn: str = field(default=None, kw_only=True)
+    region_name: str = field(default="us-east-1", kw_only=True)
```

Comment:
How do secret ARNs work here? Can we swap both of those out with session? (similar to how we [do it](https://github.com/griptape-ai/griptape-tools/blob/main/griptape/tools/base_aws_client.py#L11) in tools)

Reply:
A SecretArn can be provided which hosts the Database credentials. That is separate from the AWS credentials for calling the Redshift Data API via boto. Yes, we can swap out `region_name` and just use session, but keep `secret_arn` as an optional parameter.

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/redshift-data/client/execute_statement.html#RedshiftDataAPIService.Client.execute_statement